server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -78,7 +78,10 @@ def add_to_parser(cls, subparsers, invocation_function):
 
     @staticmethod
     def _load_data(data):
-        return sc.read(os.path.join(data, "data.h5ad"))
+        # See https://scanpy.readthedocs.io/en/latest/api/scanpy.api.read.html
+        # Based upon this advice, setting both backend= and cache= parameters.
+        # return sc.read(os.path.join(data, "data.h5ad"), cache = True)
+        return sc.read(os.path.join(data, "data.h5ad"), backed='r', cache=True)
```

Comment:
I didn't quite understand the difference between `r` and `r+` options for `backed`. Does r+ mean if you wanted to save changes back?